e2e/libs/host/harvester.py

```
@@ -84,6 +84,8 @@
             time.sleep(self.retry_interval)
         assert stopped, f"Expected vm {vm_id} to be stopped but it's not"
 
+        self.node.wait_for_node_down(power_off_node_name)
```

Ruff (0.8.0)

Undefined name `power_off_node_name`

(F821)